src/sources/http.rs

```diff
@@ -366,8 +372,8 @@ mod tests {
 
         let (rx, addr) = source(Encoding::Json, vec![]).await;
 
-        assert_eq!(200, send(addr, r#"[{"key":"value"}]"#).await);
-        assert_eq!(200, send(addr, r#"{"key2":"value2"}"#).await);
+        assert_eq!(200, send(addr, r#"[{"Log":{"key":"value"}}]"#).await);
+        assert_eq!(200, send(addr, r#"{"Log":{"key2":"value2"}}"#).await);
```

Comment:
So this seems like it'd be a breaking change.

Reply:
It would yeah. That's why the `metrics_plus_mode` existed to begin with.

Reply:
Yeah, then I almost lean towards punting this until we have some greater RFC/architecture work for specialized handling of this kind of thing. I'm sorry. :(

Does the `vector` source/sink not work for you?

Reply:
Unfortunately for my use case I really do need `http`. Infrastructure reasons.

I suppose I can just run my own fork until these discussions happen.